public/blog/2023-11-dask-survival-analysis/2023-11-dask-survival-analysis.md

```diff
@@ -0,0 +1,48 @@
+thumbImage: "orange-not-responding.png"
+frontPageImage: "orange-not-responding.png"
+blog: ["dask", "survival analysis"]
+shortExcerpt: "Sidestep the limitations of web-based visual anlaytics tools and and steer your survival analysis with confidence and ease. Dive into the entire TCGA cohort within a single Orange workflow."
+longExcerpt: "Sidestep the limitations of web-based visual anlaytics tools and and steer your survival analysis with confidence and ease. Dive into the entire TCGA cohort within a single Orange workflow."
+---
+
+Since its inception, The Cancer Genome Atlas (TCGA) project has been a treasure trove of data for biomedical researchers. The data TCGA provides is essential for researchers who want to analyze potential biomarkers. But there's a hitch. Not every researcher is a data whisperer.
+
+Luckily, there are several web-based tools that researchers can add to their toolkits. These tools make it easy to access and analyze survival data from TCGA. Notable examples are [Kaplan-Meier Plotter](https://kmplot.com/analysis/) and [GEPIA](http://gepia.cancer-pku.cn/), among [others](https://www.frontiersin.org/journals/oncology/articles/10.3389/fonc.2020.00068/full).
+
+<WindowScreenshot src="gepia-example.png" />
+
+In their very essence, these tools allow you to enter a gene name and output the results of the analysis on pre-curated TCGA datasets. For example, in the image above, I have entered the gene name TP53 and selected the cancer type lower-grade glioma (TCGA-LGG). The output is the Kaplan-Meier plot for the high- and low-risk patient groups. I can also see the p-value of the log-rank test, which tells me how significant the difference in survival between the groups is. For a detailed description of functionalities, I suggest you read this review paper by [Zheng et al.](https://www.frontiersin.org/journals/oncology/articles/10.3389/fonc.2020.00068/full).
+
+
+Perfect, this is just what I need. So where is the problem? Well, as it turns out, you can't just blindly trust the underlying source of survival data of those tools. Curated datasets between tools can vary (more in [this paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8726696/) by Idogawa et al.). And this is no small thing. How can you trust the analysis if you can trust the data source? Adding fuel to the fire here, I will let the reader guess how many of those tools are open-sourced.
+
+Ok, don't get me wrong. The utility of such tools is great despite the limitations I mentioned. However, I would like to analyze data from sources I trust. For that, I have prepared [notebooks](https://github.com/jakakokosar/tcga-data) that I can use to get gene expression and corresponding survival data for all cancer types in the TCGA project. I see the irony here. I am not asking you to trust my code. For those who read the code, you verify and replicate it on your own. For others, use the official [GDC](https://portal.gdc.cancer.gov/) data portal to download data.
```

Comment:
Ok, don't get me wrong. The utility of such tools is great despite the limitations I mentioned. -> Still, despite their limitations, these tools are useful.

Reply:
However, I would like to analyze data from sources I trust. For that, I have prepared [notebooks](https://github.com/jakakokosar/tcga-data) that I can use to get gene expression and corresponding survival data for all cancer types in the TCGA project. I see the irony here. I am not asking you to trust my code. For those who read the code, you verify and replicate it on your own. For others, use the official [GDC](https://portal.gdc.cancer.gov/) data portal to download data.
->
For that, I have prepared [notebooks](https://github.com/jakakokosar/tcga-data) that can download gene expression and survival data for all cancer types in the TCGA project. I see the irony here: I am not asking you to trust [my code](https://github.com/jakakokosar/tcga-data); if you can, verify it. Or use the official [GDC data portal](https://portal.gdc.cancer.gov/) to download data.